packages/tuono/src/ssr/index.ts

```diff
@@ -0,0 +1,4 @@
+export { MessageChannelPolyfill } from './polyfills/MessageChannel'
+export { EventPolyfill } from './polyfills/Event'
+export { MessageEventPolyfill } from './polyfills/MessageEvent'
+export { serverSideRendering } from './server'
```

Comment:
```suggestion
export { serverSideRendering } from './server'

export { MessageChannelPolyfill } from './polyfills/MessageChannel'
export { EventPolyfill } from './polyfills/Event'
export { MessageEventPolyfill } from './polyfills/MessageEvent'
```

Style: just reorder import adding groups and putting `serverSideRendering` as first

Reply:
What about adding a formatting rule for this?

Reply:
I'm not aware of any way to statically define and organise exports via eslint / prettier. 
Those kind of rules are available for import, not for exports

Reply:
How about this one? 

https://www.npmjs.com/package/eslint-plugin-sort-exports

Reply:
I have seen it, but I haven't proposed it because:
- looks un-maintened (no commit in nearly 1 year) 
- examples refer only to legacy config and not to flat config

If you want I can give it a try anyway 
(since it exports only one rule there should be no problem using in with the flat config)

Reply:
Yeah I think it would be worth giving it a try. 
In the end it is just matter of reorder the exports. 
In the download chart sounds pretty used still

Reply:
Ok, I'll take a look at it tomorrow, meanwhile I'll go ahead and manually apply the fix so we can merge this.